README.md

```diff
@@ -112,26 +125,27 @@ $ nix-env -qa --meta --json '.*' >meta.json
 ```
 Then, run `sbomnix` with `--meta` argument to tell sbomnix to read meta information from the given json file:
 ```bash
-$ sbomnix /nix/store/8nbv1drmvh588pwiwsxa47iprzlgwx6j-wget-1.21.3 --meta meta.json
+$ nix run .#sbomnix -- /nix/store/8nbv1drmvh588pwiwsxa47iprzlgwx6j-wget-1.21.3 --meta meta.json
 ```
 
 #### Generate SBOM Including Buildtime Dependencies
 By default `sbomnix` scans the given target for runtime dependencies. You can tell sbomnix to determine the buildtime dependencies using the `--type` argument. 
-Acceptable values for `--type` are `runtime, buildtime, both`. Below example generates SBOM including buildtime dependencies. Notice that determining buildtime dependencies does not require realising (building) the target.
+Acceptable values for `--type` are `runtime, buildtime, both`. Below example generates SBOM including buildtime dependencies.
+Notice: as opposed to runtime dependencies, determining the buildtime dependencies does not require building the target.
 ```bash
-$ sbomnix /nix/store/8nbv1drmvh588pwiwsxa47iprzlgwx6j-wget-1.21.3 --meta meta.json --type=buildtime
+$ nix run .#sbomnix -- /nix/store/8nbv1drmvh588pwiwsxa47iprzlgwx6j-wget-1.21.3 --meta meta.json --type=buildtime
 ```
 #### Generate SBOM Based on Result Symlink
 `sbomnix` can be used with output paths too (e.g. anything which produces a result symlink):
 ```bash
-$ sbomnix /path/to/result 
+$ nix run .#sbomnix -- /path/to/result 
```

Comment:
This should stay as-is, see above comment.

Reply:
Fixed: https://github.com/tiiuae/sbomnix/compare/2a1f91600695b7261b70eb3f82ca4c437d5039ac..74f874012559a239f64f6af57d564e0869ee1554